src/ontoweaver/base.py

```diff
@@ -287,6 +291,30 @@ def valid(self, val):
             return False
         return True
 
+    def __repr__(self):
+        if hasattr(self, "from_subject"):
+            from_subject = self.from_subject
+        else:
+            from_subject = "."
+
+        if self.target:
+            target_name = self.target.__name__
+        else:
+            target_name = "."
+
+        if self.edge:
+            edge_name = self.edge.__name__
+        else:
+            edge_name = "."
+
+        if self.properties_of:
+            props = self.properties_of
+        else:
+            props = "{}"
+
+        params = {k:v for k,v in self.parameters.items() if k not in ['subclass', 'from_subject']}
+
+        return f"<Transformer/{type(self).__name__}{params} {self.columns} => [{from_subject}]--({edge_name})->[{target_name}/{props}]>"
```

Comment:
This sometime results in weird logging outputs in case some values are` None`, for example:

`Calling transformer <Transformer/map{} ['hugoSymbol'] => [.]--(variant_in_gene)->[gene_hugo/{<Transformer/map{} ['entrezGeneId'] => [.]--(.)->[./{}]>: 'entrez_gene_id', <Transformer/map{} ['geneSummary'] => [.]--(.)->[./{}]>: 'gene_summary'}]>:`